src/Infrastructure/Processor/Process/MediaTranslationShopware6ExportProcess.php

```diff
@@ -0,0 +1,150 @@
+        $altTranslations = $multimedia->getAlt()->getTranslations();
+        $titleTranslations = $multimedia->getTitle()->getTranslations();
+        $languages = array_unique(array_merge(array_keys($altTranslations), array_keys($titleTranslations)));
+        $ergoTranslations = [];
+        foreach ($channel->getLanguages() as $channelLanguage) {
+            if (!in_array($channelLanguage->getCode(), $languages)) {
+                continue;
+            }
+            if ($this->languageRepository->exists($channel->getId(), $channelLanguage->getCode())) {
+                $shopwareLanguage = $this->languageRepository->load($channel->getId(), $channelLanguage->getCode());
+                Assert::notNull(
+                    $shopwareLanguage,
+                    sprintf('Expected a value other than null for product lang  %s', $channelLanguage->getCode())
+                );
+                $ergoTranslations[] = new Shopware6MediaTranslation(
+                    null,
+                    $altTranslations[$shopwareLanguage->getIso()] ?? null,
+                    $titleTranslations[$shopwareLanguage->getIso()] ?? null,
+                    $shopwareLanguage->getId()
+                );
+            }
+        }
+        $shopware6Media->updateTranslated($ergoTranslations);
+
+        if ($shopware6Media->isModified()) {
+            $this->logger->info(
+                'Updating multimedia media translations start',
+                [
+                    'multimediaId' => $multimedia->getId()->getValue(),
+                    'multimediaName' => $multimedia->getName(),
+                    'mediaId' => $shopware6Media->getId()
+                ]
+            );
+            $this->updateTranslations($channel, $shopware6Media);
+            $this->logger->info(
+                'Updating multimedia media translations end',
+                [
+                    'multimediaId' => $multimedia->getId()->getValue(),
+                    'multimediaName' => $multimedia->getName(),
+                    'mediaId' => $shopware6Media->getId()
+                ]
+            );
+        }
+    }
+
+    /**
+     * @throws GuzzleException
+     */
+    private function updateTranslations(Shopware6Channel $channel, Shopware6Media $shopware6Media): void
+    {
+        if (!empty($shopware6Media->getTranslations())) {
+            $action = new PatchMediaAction($shopware6Media);
+            $this->connector->execute($channel, $action);
+        }
+    }
+
+    /**
+     * @return Shopware6MediaTranslation[]
+     * @throws GuzzleException
+     */
+    private function getMediaTranslations(Shopware6Channel $channel, string $shopwareMediaId): array
+    {
+        $query = new Shopware6QueryBuilder();
+        $query->limit(1000);
+
+        $action = new GetMediaTranslations($query, $shopwareMediaId);
+
+        try {
+            return $this->connector->execute($channel, $action);
+        } catch (ClientException $exception) {
```

Comment:
why we ignore this exception?

Reply:
thank, you - fixed - passed to export errors